grizzly/testdata/variables/messagequeue.py

```diff
@@ -126,7 +129,7 @@ class AtomicMessageQueue(AtomicVariable[str]):
     _zmq_url = 'tcp://127.0.0.1:5554'
     _zmq_context: zmq.Context
 
-    arguments: Dict[str, Any] = {'repeat': bool_typed, 'url': str, 'wait': int}
+    arguments: Dict[str, Any] = {'content_type': str, 'repeat': bool_typed, 'url': str, 'wait': int}
```

Comment:
validation of provided `content_type`?
type should be something like `TransformerContentType.from_string`.